danmaku/ui/commonMain/FixedDanmakuTrack.kt

```diff
@@ -0,0 +1,105 @@
+package me.him188.ani.danmaku.ui
+
+import androidx.compose.runtime.IntState
+import androidx.compose.runtime.LongState
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+
+/**
+ * FixedDanmakuTrack 中的弹幕在以下情况会移除:
+ * - tick 中的逻辑帧检测
+ * - [FixedDanmakuTrack.place] 覆盖了正在显示的弹幕
+ * - 调用 [DanmakuTrack.clearAll]
+ * 移除时必须调用 [DanmakuTrack.onRemoveDanmaku] 避免内存泄露.
+ */
+@Stable
+class FixedDanmakuTrack(
+    val trackIndex: Int,
+    val fromBottom: Boolean,
+    override val frameTimeNanos: LongState,
+    private val trackHeight: IntState,
+    private val screenWidth: IntState,
+    private val screenHeight: IntState,
```

Comment:
should test `place` & `tryPlace` if `screenWidth` and/or `screenHeight` and/or `trackHeight` are 0

(state test, not ui test)